app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  has_many :tweets
+  has_secure_password
+
```

Comment:
Resolved - Added validation to only allow unique username and email address to create new User and persist to database. However it seems as if I should also let the user know what happened, like show an error message to them, informing that "name is taken try again"